examples/hosts/app-integration/external-data/src/view/taskListView.tsx

```diff
@@ -94,17 +92,21 @@ const TaskRow: React.FC<ITaskRowProps> = (props: ITaskRowProps) => {
 				></input>
 			</td>
 			<td>
-				<button onClick={deleteTask} style={{ background: "none", border: "none" }}>
+				<button onClick={deleteDraftTask} style={{ background: "none", border: "none" }}>
 					❌
 				</button>
 			</td>
-			<td style={{ visibility: showName, backgroundColor: diffColor }}>{incomingName}</td>
-			<td style={{ visibility: showPriority, backgroundColor: diffColor }}>
-				{incomingPriority}
-			</td>
+			{showNameDiff && (
+				<td style={{ backgroundColor: diffColor }}>{externalDataSnapshot.name}</td>
+			)}
+			{showPriorityDiff && (
+				<td style={{ backgroundColor: diffColor, width: "30px" }}>
+					{externalDataSnapshot.priority}
+				</td>
+			)}
```

Comment:
Not a React expert, just pointing out this syntax weirds me out 😄 .

Reply:
the && syntax effectively doesn't render the element at all, whereas the visibility does render but leaves it invisible, leaving this weird gap in the middle.
![image](https://user-images.githubusercontent.com/6777404/221319809-7d89531a-0bb5-46a5-8b20-09790e4fc8c4.png)
vs
![image](https://user-images.githubusercontent.com/6777404/221319615-23aae98d-e159-4a83-847a-0f1fdbc2c5f1.png)

The && syntax is a bit old school I'll admit, but I don't believe it's "wrong" yet technically :p 
Since it's not the main event here I'm okay with this syntax, but can also change it to the render but keep it hidden paradigm :p